Userland/Services/WindowServer/Window.cpp

```diff
@@ -11,6 +11,9 @@
 #include "ConnectionFromClient.h"
 #include "Event.h"
 #include "EventLoop.h"
+#include "LibCore/Account.h"
+#include "LibCore/ProcessStatisticsReader.h"
+#include "LibCore/SessionManagement.h"
```

Comment:
These should be included using `<>` instead of `""`, and properly sorted afterwards.